akka-http-zio-json/src/main/scala/de/heikoseeberger/akkahttpziojson/ZioJsonSupport.scala

```diff
@@ -102,7 +102,7 @@ trait ZioJsonSupport {
     Unmarshaller(_ =>
       bs => {
         val decoded = jd.decodeJsonStreamInput(ZStream.fromIterable(bs))
-        Unsafe.unsafeCompat(implicit u => rt.unsafe.runToFuture(decoded))
+        Unsafe.unsafe(implicit u => rt.unsafe.runToFuture(decoded))
```

Comment:
@gabriele83 I'm interested in using your changes in https://github.com/pjfanning/pekko-http-json.

Could you help by explaining why this ZIO change is useful? The rest of the PR is about Circe but I'd like to apply this change to pekko-http too if it is an improvement.